.github/workflows/lint-python.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
       - run: flake8 . --count --exit-zero --max-complexity=29 --max-line-length=167 --show-source --statistics
       - run: isort -c .
-      - run: pip install -e . # editable mode needs build backend support for PEP660 (setuptools >= 64)
+      - run: pip install -editable .  # will require build backend support for PEP660 (setuptools >= 64)
```

Comment:
```suggestion
      - run: pip install —editable .  # will require build backend support for PEP660 (setuptools >= 64)
```

Reply:
Done

Have also removed requirements*.txt and used relevant sections in pyproject.toml for various pip installs.
I've segregated build, test and code dependencies, but you can cut and dice it any way you prefer.